src/renderers/WebGLRenderer.js

```diff
@@ -1063,13 +1065,19 @@ function WebGLRenderer( parameters ) {
 
 					for ( var i = 0; i < object.material.length; i ++ ) {
 
-						initMaterial( object.material[ i ], scene, object );
+						if ( ! visited[ object.material[ i ].uuid ] ) {
```

Comment:
`if ( object.material[ i ].uuid in compiled ) {`

Reply:
...not. :wink: